app.test.js

```diff
@@ -0,0 +1,129 @@
+const request = require('supertest');
+const app = require('./app');
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('./knexfile')[environment]
+const database = require('knex')(configuration);
+
+
+
+describe('API', () => {
```

Comment:
Thanks so much for the thorough testing!

Reply:
Thanks, @jacquelinebelle for reviewing :)